tests/integration_test/src/tests/seccomp/mod.rs

```diff
@@ -0,0 +1,49 @@
+use oci_spec::runtime::{
+    Arch, LinuxBuilder, LinuxSeccomp, LinuxSeccompAction, LinuxSeccompBuilder, LinuxSyscallBuilder,
+    ProcessBuilder, Spec, SpecBuilder,
+};
+use test_framework::{Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+
+fn create_spec(seccomp: LinuxSeccomp) -> Spec {
+    SpecBuilder::default()
+        .linux(
+            LinuxBuilder::default()
+                .seccomp(seccomp)
+                .build()
+                .expect("error in building linux config"),
+        )
+        .process(
+            ProcessBuilder::default()
+                .args(vec!["runtimetest".to_string(), "seccomp".to_string()])
+                .build()
+                .expect("error in creating process config"),
+        )
+        .build()
+        .unwrap()
+}
+
+fn seccomp_test() -> TestResult {
+    let spec = create_spec(
+        LinuxSeccompBuilder::default()
+            .default_action(LinuxSeccompAction::ScmpActAllow)
+            .architectures(vec![Arch::ScmpArchX86_64])
+            .syscalls(vec![LinuxSyscallBuilder::default()
+                .names(vec![String::from("getcwd")])
+                .action(LinuxSeccompAction::ScmpActErrno)
+                .build()
+                .unwrap()])
+            .build()
+            .unwrap(),
+    );
```

Comment:
Do we not need the iterating check like https://github.com/opencontainers/runtime-tools/blob/master/cmd/runtimetest/main.go#L926-L940 , where we specifically check if the syscall name is `getcwd` and only ?

My thought is because we have setup where we specifically run a runtimetest test according to the commandline arg provided to it, we do not need that. Can you confirm that it is so?

Reply:
I think your thought is correct.

Based on https://github.com/opencontainers/runtime-tools/blob/master/cmd/runtimetest/main.go#L1253, the `runtimetest` executable will execute all validators regardless of the specific integration test case. Therefore, it needs to check if the syscall name is `getcwd`.
We don't need that check because the `validate_seccomp` function will only execute in the `seccomp` test case.

Reply:
Yep, thanks!